Add scatter plot functionality

avonow opened the pull request:
Adds the option to create a scatter plot.

avonow committed:
Add scatter plot functionality

Adds the option to create a scatter plot.

huntc reviewed (commented):
Thanks for this!

Would it be possible to provide an example of scatter in use (see how examples are structured…)?

huntc commented on src/horizontal_series.rs:
```diff
@@ -119,17 +120,26 @@ impl HorizontalSeries {
                             html!(<polyline points={points} class={classes.to_owned()} fill="none"/>),
                         );
-                    } else {
+                    } else if props.series_type == SeriesType::Bar {
```
Cargo clippy should probably highlight that we need a match statement now…

avonow replied:
Updated to use match statements instead. (Though, Cargo clippy didn't highlight the need for a match; Rory also found this on the code.)

avonow committed:
Match implementation and updated README

If/else statements changed to match, and README.md updated with new images.

avonow commented:
In the last commit here, I've added images from my scatter implementation and updated the README. I changed the layout slightly, so that the images will be at 70% and centred. If this is not desirable, very happy to revert.

avonow commented:
We've been informed that the `data_labels` property can be used for scatter plots, so a separate PR will be raised to just document that with an example, instead of this current PR.

huntc commented:
> We've been informed that the `data_labels` property can be used for scatter plots, so a separate PR will be raised to just document that with an example, instead of this current PR.

Thanks so much. That'd be wonderful.
